etc/profile-m-z/telegram.profile

```diff
@@ -41,7 +45,7 @@ seccomp.block-secondary
 shell none
 
 disable-mnt
-private-bin telegram,Telegram,telegram-desktop
+private-bin bash,telegram,Telegram,telegram-desktop,sh
```

Comment:
add xdg-open too and respect ordering:
private-bin bash,sh,telegram,Telegram,telegram-desktop,xdg-open